browser/src/control/jsdialog/Util.KeyboardGridNavigation.ts

```diff
@@ -0,0 +1,166 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * JSDialog.ScrollableBar - helper for creating toolbars with scrolling left/right
+ */
+
+/* global JSDialog $ */
+
+// declare var JSDialog: any;
+
+function moveFocus(
+	parentContainer: HTMLElement,
+	currentElement: HTMLElement,
+	direction: 'next' | 'previous',
+	axis: 'horizontal' | 'vertical',
+	nextElement?: Element,
+) {
+	const focusableElements = Array.from(
```

Comment:
we should use single function  everywhere for getting focusable elements, there is already util for that:
https://github.com/CollaboraOnline/online/blob/f55c1a1403fcd350e81fdb98bd6c3a6b05cfda2b/browser/src/control/jsdialog/Util.FocusCycle.js#L18